<!--StartFragment-->

Princess Beatrice Sunday- Jimmy

**RNA Sequencing**

**Introduction**

The study of gene expression began in the early 1970s with the northern blot technique. This method had limitations, leading to quantitative PCR (qPCR) development in the 1980s and microarrays in the 1990s (Barde _et al_.,2021). Despite these advancements, challenges remained. RNA sequencing (RNA-Seq) emerged as a new technique that uses Next-Generation Sequencing (NGS) to understand changes in gene expression or novel splicing events. It is used for transcriptome analysis and provides far more precise measurements of levels of transcripts and their isoforms than the previous methods used in transcriptome profiling (Ozsolak and Milos, 2011).

**Methods/Workflow of RNA Sequencing**

The RNA sequencing workflow starts with RNA extraction, noting that single-stranded RNA is less stable than double-stranded DNA. After extraction, RNA is converted into complementary DNA (cDNA). Library preparation involves fragmenting the RNA or cDNA sample into smaller pieces and attaching specialized adapters to these fragments. Sequencing of the fragments is carried out using high-throughput technologies. Data analysis is then performed to interpret the sequencing results and identify gene expression patterns and transcript variants (Di _et al_., 2020)

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXcq6kziiq-hp4V4aMRyPs6xkAPxxQO00op8V1Imsn2Bt8jCaHg23VWdxjRIR2jVaSJXmjb9LByd3sGVSFYO1pkyHdfilNrgNx88lynyIa1MrYc9idXtYMNsnubu49gCQzx2TvBH4e9w9FYEnGhtcL-BOYw?key=PGEwbLY1AquH_2cVR6_VLg)

Figure 1: A schematic diagram showing the workflow of RNA-Seq (Dahal, 2023).

**Importance/Relevance of RNA Sequencing in Cancer**

RNA sequencing (RNA-seq) is a crucial tool in cancer research and treatment, offering detailed insights into gene expression and function. It enables the identification of genes associated with cancer, including dysregulated non-coding RNAs, which can aid in early cancer detection. RNA-seq supports the discovery of cancer biomarkers and the development of targeted therapies, providing higher sensitivity and accuracy than traditional methods like microarrays (Hong _et al_., 2020). By analyzing gene expression patterns across tissues and detecting low-abundance transcripts, RNA-seq helps researchers understand tumor biology, identify oncogenes and tumor suppressor genes, and explore tumor heterogeneity. This comprehensive approach is invaluable for developing personalized treatment strategies and advancing RNA-based therapeutics(Zhu _et al_., 2023).

**Application of RNA Sequencing**

RNA sequencing has been applied in various biomedical fields, such as precision medicine and projects. For instance, RNA-Seq was utilized in the Cancer Genome Atlas project, which began in 2005. This sequencing technique analyzed thousands of tumor samples, enhancing the understanding of cancer progression (Lee,2016). Additionally, the ENCODE project helped identify functional elements in the human genome, offering insights into genetic and epigenetic regulation(Luo _et al_., 2020).

**References**

Barde, L. Y., Adamu, H., Uzoma, G. I., Bello, M., and Danjuma, M. A. (2021). Gene expression and its application in biotechnology. _GSC Advanced Research and Reviews, 8_(1), 012–021.[ https://doi.org/10.30574/gscarr.2021.8.1.0093](https://doi.org/10.30574/gscarr.2021.8.1.0093).

Dahal, P. (2023). RNA sequencing: Definition, principle, steps, types, uses. Microbe Notes. Accessed on September 3rd, 2024. <https://microbenotes.com/rna-sequencing-principle-steps-types-uses/>. 

Di, L., Fu, Y., Sun, Y., Li, J., Liu, L., Yao, J., Wang, G., Wu, Y., Lao, K., Lee, R. W., Zheng, G., Xu, J., Oh, J., Wang, D., Xie, X. S., Huang, Y., and Wang, J. (2020). RNA sequencing by direct tagmentation of RNA/DNA hybrids. _Proceedings of the National Academy of Sciences of the United States of America, 117_(6), 2886–2893. <https://doi.org/10.1073/pnas.1919800117>. 

Hong, M., Tao, S., Zhang, L., Diao, L.-T., Huang, X., Huang, S., Xie, S.-J., Xiao, Z.-D., and Zhang, H. (2020). RNA sequencing: New technologies and applications in cancer research. _Journal of Hematology and Oncology, 13_, 166.<https://doi.org/10.1186/s13045-020-01005-x>. 

Lee, J. S. (2016). Exploring cancer genomic data from the Cancer Genome Atlas project. _BMB Reports, 49_(11), 607–611. <https://doi.org/10.5483/bmbrep.2016.49.11.145>. 

Luo, Y., Hitz, B. C., Gabdank, I., Hilton, J. A., Kagda, M. S., Lam, B., Myers, Z., Sud, P., Jou, J., Lin, K., Baymuradov, U. K., Graham, K., Litton, C., Miyasato, S. R., Strattan, J. S., Jolanki, O., Lee, J.-W., Tanaka, F. Y., Adenekan, P., O’Neill, E., & Cherry, J. M. (2020). New developments on the Encyclopedia of DNA Elements (ENCODE) data portal. _Nucleic Acids Research, 48_(D1), D882–D889.[ https://doi.org/10.1093/nar/gkz1062](https://doi.org/10.1093/nar/gkz1062)

Ozsolak, F., and Milos, P. M. (2011). RNA sequencing: Advances, challenges, and opportunities. _Nature Reviews Genetics, 12_(2), 87–98. <https://doi.org/10.1038/nrg2934>. 

Zhu, Z., Jiang, L., and Ding, X. (2023). Advancing breast cancer heterogeneity analysis: Insights from genomics, transcriptomics, and proteomics at bulk and single-cell levels. _Cancers (Basel), 15_(16), 4164. <https://doi.org/10.3390/cancers15164164>.

\


<!--EndFragment-->
